rotkehlchen/chain/solana/decoding/decoder.py

```diff
@@ -17,3 +37,72 @@ def __add__(self, other: 'SolanaDecodingRules') -> 'SolanaDecodingRules':
         return SolanaDecodingRules(
             address_mappings=self.address_mappings | other.address_mappings,
         )
+
+
+class SolanaTransactionContext(NamedTuple):
+    transaction: SolanaTransaction
```

Comment:
is this needed? you can just pass the `SolanaTransaction` directly as the context.